apps/passport-server/migrations/47_bot_forwarding.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE telegram_forwarding (
```

Comment:
it appears as though we really only need two columns:

`sender_topic_id` which is nullable and a foreign key to telegram_topics
`recipient_topic_id` which is nullable and a foreign key to telegram_topics

and then a unique index on (`sender_topic_id`, `recipient_topic_id`)

Reply:
use nulls not distinct to handle the nullable case

Reply:
Done